src/test/java/CarTest.java

```diff
@@ -1,24 +1,23 @@
-import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
 
-import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.Assertions.*;
 
 class CarTest {
-    private Car car;
-
-    @BeforeEach
-    void setUp() {
-        car = new Car();
-    }
-
     @ParameterizedTest
     @CsvSource({"0,0", "3,0", "4,1", "9,1"})
     @DisplayName("4 이상의 숫자를 입력하면 1칸, 4 미만의 숫자를 입력하면 0칸 전진한다.")
     void moveTest(int number, int expectedLocation) {
+        final Car car = new Car("pobi");
         car.makeMove(number);
-        assertThat(car.currentLocation()).isEqualTo(expectedLocation);
+        assertThat(car.location()).isEqualTo(new Location(expectedLocation));
```

Comment:
```suggestion
        assertThat(car).isEqualTo(new Car("pobi", expectedLocation));
```
값을 꺼내기 보다 위와 같이 객체로 비교할 수 있도록 의식적인 연습을 해본다.
객체로 비교하기 힘든 경우에만 값을 꺼내 비교한다.